jenkins/job-configs/global.yaml

```diff
@@ -35,7 +35,7 @@
             script-only-if-failed: False
         # Use the plugin for the build log, since it isn't available on Jenkins slaves.
         - google-cloud-storage:
-            credentials-id: kubernetes-jenkins
+            credentials-id: '{credentials-id}'
             uploads:
                 - build-log:
                     log-name: build-log.txt
```

Comment:
Oh, right. Rather than templating it for both PR and post-commit Jenkins I'll just copy this bit. Eventually we should stop using this plugin altogether.